Documentation/concepts/ipam/index.rst

```diff
@@ -12,6 +12,16 @@ Address Management
 
 Cilium supports multiple different address management modes:
 
+Cilium Container Networking Control Flow
```

Comment:
This paragraph has been inserted in between the sentence above which points out the different modes, and the toctree below which lists the modes with links. Did you mean to put this new section above or below the existing content?

Reply:
I have addressed your comments